src/runtime/audience-action-flow.js

```diff
@@ -344,6 +344,12 @@ export class AudienceActionFlow {
       };
       this.deps_.eventManager().logEvent(event, eventParams);
     });
+    this.deps_.eventManager().logEvent({
```

Comment:
(Optional) What about logging this `handleSurveyDataTransferRequest_` where failure is logged to make it clear that either of them is logged?

Reply:
Will we be able to distinguish between success and failures in the logs? Or are we not concerned with that?

Reply:
I meant putting it in the `else` statement of `if (!gaLoggingSuccess)` so that either of failure or success is logged.

Reply:
Ah I see, yes I can do that.